infrastructure/helm/quantumserverless/charts/jupyter/values.yaml

```diff
@@ -5,6 +5,7 @@
 jupyterToken: "123"
 managerHost: ""
 managerPort: 5000
+GATEWAY_HOST: http://gateway:8000
```

Comment:
Should this live inside `deloyment.yml` in env list?

```yml
- name: GATEWAY_HOST
  value: {{ .Values.gatewayHost }}
```

Reply:
yes